app/endpoints/raffle.py

```diff
@@ -305,6 +305,58 @@ async def get_tickets(
     return tickets
 
 
+@router.post(
+    "/tombola/tickets/add/{user_id}/{type_id}",
+    response_model=list[schemas_raffle.TicketComplete],
+    status_code=201,
+    tags=[Tags.raffle],
+)
+async def add_ticket_to_user(
+    user_id: str,
+    type_id: str,
+    db: AsyncSession = Depends(get_db),
+    redis_client: Redis | None = Depends(get_redis_client),
+    user: models_core.CoreUser = Depends(is_user_a_member_of(GroupType.soli)),
+    request_id: str = Depends(get_request_id),
+):
+    """
+    Add a ticket to another user
+
+    **The user must be a member of the group soli to use this endpoint**
+    """
+    type_ticket = await cruds_raffle.get_typeticket_by_id(typeticket_id=type_id, db=db)
+    if type_ticket is None:
+        raise ValueError("Bad typeticket association")
```

Comment:
You should raise an `HTTPException` to prevent Hyperion from crashing.

Reply:
Fixed with a 404 error
Fixed also at line 378